### PR TITLE
Add latest tag to API docker image

### PR DESCRIPTION
**Pull request by floreks**
**What this PR does / why we need it**:
As we need access to latest API image for Dashboard e2e tests, I'd like to add latest tag that will be updated with every build.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: api/hack/ci/ci-deploy-dev-cloud.sh [hunk 1]
```diff
@@ -17,6 +17,7 @@ echodate "Successfully finished building binaries"
 
 echodate "Building and pushing quay images"
 retry 5 ./api/hack/push_image.sh $GIT_HEAD_HASH $(git tag -l --points-at HEAD)
+retry 5 ./api/hack/push_image.sh $GIT_HEAD_HASH "latest"
```
Comment (mrIncompetent) [inline api/hack/ci/ci-deploy-dev-cloud.sh]: You can append the tag to the line above.
The script uses all supplied arguments as tags

Comment (floreks) [reply api/hack/ci/ci-deploy-dev-cloud.sh]: Perfect.

**Comment by floreks** [comment]
Done. PTAL

**Pull request merged**